### PR TITLE
vue runtime warning fixes

### PR DESCRIPTION
**Pull request by bafolts**
With the `development` flag set on webpack warnings are logged to the console during runtime for vue typing errors. Fixes included here.

### CODE REVIEW

File: src/components/CorporationsFilter.ts [hunk 1]
```diff
@@ -150,7 +150,6 @@ export const CorporationsFilter = Vue.component('corporations-filter', {
         </div>
         <div class="corporations-filter-group" v-for="entry in corpsByModule">
             <div class="corporations-filter-toolbox-cont">
-                <h2>{{ name }}</h2>
```
Comment (bafolts) [inline src/components/CorporationsFilter.ts:153]: Nothing is using this as far as I can tell. Vue logs warning that `name` is not a provided prop or data.

File: src/components/GameEnd.ts [hunk 2]
```diff
@@ -96,7 +96,7 @@ export const GameEnd = Vue.component('game-end', {
                     </a>
                 </div>
                 <div v-if="!isSoloGame() || player.isSoloModeWin" class="game-end-winer-announcement">
-                    <span v-for="p in getWinners()"><log-player :class="getEndGamePlayerHighlightColorClass(p.color)">{{ p.name }}</log-player></span> won!
+                    <span v-for="p in getWinners()"><span :class="'log-player ' + getEndGamePlayerHighlightColorClass(p.color)">{{ p.name }}</span></span> won!
```
Comment (bafolts) [inline src/components/GameEnd.ts:99]: The usage of `log-player` and `log-card` was creating a custom HTML element that we never defined. Vue was warning against this.

File: src/components/card/CardRowComponent.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ export const CardRowComponent = Vue.component('CardRowComponent', {
   },
   template: ` 
         <CardRenderItemComponent v-if="isItem()" :item="data"/>
-        <CardProductionBoxComponent v-else-if="isProduction()" :data="data.rows" />
+        <CardProductionBoxComponent v-else-if="isProduction()" :rows="data.rows" />
```
Comment (bafolts) [inline src/components/card/CardRowComponent.ts:29]: I changed this name from `data` to `rows` as vue has it's own `data` property on the component which could become confusing.

File: src/styles/log.less [hunk 4]
```diff
@@ -47,6 +47,3 @@ log-card {
     text-transform: uppercase;
 }
 
-log-generation {
```
Comment (bafolts) [inline src/styles/log.less:50]: Nothing using this as far as I can tell.

**Review by kberg — APPROVED** [review]

**Pull request merged**